setup.cfg

```diff
@@ -20,4 +16,10 @@ ignore =
     *.egg-info/*
 ignore-default-rules = true
 ignore-bad-ideas =
-    tests/pkgs/localeapp/*
+    tests/pkgs/localeapp/**/*.mo
+
+[tool:pytest]
+python_files = test_*.py
+testpaths =
+    tests
+addopts = -W always --cov --cov-report=term-missing
```

Comment:
I tested it and I'm pretty sure you can drop the cov-report here because it uses the values from .coveragerc by default, which already have it setup how we want it.

Reply:
Done.